src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -34,7 +34,8 @@ foam.CLASS({
     { name: 'ERROR_MSG', message: 'There was a problem creating your account.' },
     { name: 'SELECTION_TEXT', message: 'Select your country' },
     { name: 'SELECTION', message: 'Please select...' },
-    { name: 'VALIDATION_ERR_TEXT', message: 'Please enter job title'}
+    { name: 'VALIDATION_ERR_TEXT', message: 'Please enter job title'},
+    { name: 'DISCLAIMER', message: '*Ablii does not currently support businesses in Quebec. We are working hard to change this! If you are based in Quebec, check back for updates.' }
```

Comment:
This disclaimer references a product name and is circumstantial - please remove or add the ability to reference a disclaimer using an expression or another means to make it dynamic.

Reply:
hmmm -- Since the plan is to not have country selection till the business Registration I'm going to incorporate this there - https://nanopay.atlassian.net/browse/NP-1476